api/envoy/extensions/filters/http/header_to_metadata/v3/header_to_metadata.proto

```diff
@@ -90,25 +90,36 @@ message Config {
     option (udpa.annotations.versioning).previous_message_type =
```

Comment:
Add `// [#next-free-field: 6]` as a single line between the comment and `message Rule` (and same for the v4alpha version).